DESCRIPTION

```diff
@@ -10,36 +10,20 @@ Authors@R: c(
-    person("Caroline", "McKenna", email = "carolinemckenna101@gmail.com", role = "ctb"),
-    person("Catherine", "Park", email = "jcathyp@gmail.com", role = "ctb", comment = c(ORCID = "0000-0002-8273-9620")),
-    person("Abbie", "Benfield", email = "abbidabbers@gmail.com", role = "ctb"),
-    person("Georgia", "Gans", email = "georgiagans@live.com", role = "ctb"),
-    person("Kacey", "Jean-Jacques", email = "kjeanjacques@smith.edu", role = "ctb"),
-    person("Swaha", "Bhattacharya", email = "itsswahabhattacharya@gmail.com", role = "ctb"),
-    person("Vivian", "Almaraz", email = "viviansofia101@gmail.com", role = "ctb"),
-    person("Elle Jo", "Whalen", email = "ewhalen@smith.edu", role = "ctb"),
-    person("Jacqueline", "Chen", email = "jchen76@smith.edu", role = "ctb"),
-    person("Michelle", "Flesaker", email = "mflesaker@smith.edu", role = "ctb"),
-    person("Irene", "Foster", email = "ifoster25@smith.edu", role = "ctb"),
-    person("Aushanae", "Haller", email = "aushanaenhaller@gmail.com", role = "ctb"),
-    person("Benjamin", "Bruncati", email = "kbruncati@smith.edu", role = "ctb", comment = c(ORCID = "0000-0001-8545-5984")),
-    person("Quinn", "White", email = "quinnarlise@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5399-0237")),
-    person("Tianshu", "Zhang", email = "tzhang26@smith.edu", role = "ctb", comment = c(ORCID = "0000-0002-3004-4472")),
-    person("Katelyn", "Diaz", email = "katndiaz@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-6108-1682")),
-    person("Rose", "Porta", email = "rporta@smith.edu", role = "ctb"),
-    person("Renee", "Wu", email = "rwu30@smith.edu",role ="ctb"),
-    person("Arris", "Moise", email = "amoise@smith.edu", role = "ctb"),
-    person("Kate", "Phan", email = "kphan@smith.edu", role = "ctb"),
-    person("Grace", "Hartley", email = "grace.hartley@gmail.com", role = "ctb"),
-    person("Silas", "Weden", email = "silasweden@gmail.com", role = "ctb"),
     person("Emma", "Vejcik", email = "evejcik@gmail.com", role = "ctb", comment = c(ORCID = "0000-0001-5093-6462")),
     person("Nikki", "Schuldt", email = "nikkischuldt@gmail.com", role = "ctb"),
-    person("Tess", "Goldmann", email = "tessgoldmann@aol.com", role = "ctb")
+    person("Tess", "Goldmann", email = "tessgoldmann@aol.com", role = "ctb"),
+    person("Hongtong", "Lin", email = "cccynthialht@gmail.com", role = "ctb"),
+    person("Alejandra", "Munoz", email = "amunozgarcia@smith.edu", role = "ctb"),
+    person("Elina", "Gordon-Halpern", email = "egordonhalpern@smith.edu", role = "ctb")
     )
 Maintainer: Albert Y. Kim <albert.ys.kim@gmail.com>
-Description: Datasets and wrapper functions for tidyverse-friendly introductory linear regression, used in "Statistical Inference via Data Science: A ModernDive into R and the Tidyverse" available at <https://moderndive.com/>.
+<<<<<<< HEAD
+Description: Datasets and wrapper functions for 
+    tidyverse-friendly introductory linear regression, used 
```

Comment:
@egordonhalpern I guess I'd prefer it if we didn't do the indentation here, because then I think these changes will drop out of the PR entirely. Can you make it like it was before?